cmd/lakectl/cmd/root.go

```diff
@@ -41,7 +41,11 @@ lakectl is a CLI tool allowing exploration and manipulation of a lakeFS environm
 			return
 		}
 		if _, ok := cfgFileErr.(viper.ConfigFileNotFoundError); ok {
```

Comment:
if len(cfgFile) > 0 && errors.As(cfgFileErr, &viper.ConfigFileNotFoundError{}) {

Reply:
This is also inaccurate as it will catch the second condition for `cfgFileErr != nil`.
Refactored again.